### PR TITLE
Policies and Policies

### PR DESCRIPTION
**Pull request by mattmahoneyrh**

### CODE REVIEW

File: views/policies.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        el = self.web_driver.find_element_by_xpath("//*[contains(text(),'INCLUDES')]")
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.click()
+        self.ui_utils.sleep(2)
+        el = self.web_driver.find_element_by_id('chosen_value')
+        assert self.ui_utils.wait_until_element_displayed(el, 10)
+        el.send_keys(product_name)
+
+        # Click on Check box
+        el = self.web_driver.find_element_by_class_name('fa-check').click()
```
Comment (gbaufake) [inline views/policies.py]: @mattmahoneyrh 
Fa-check is font-awesome class.

What happens if there are more than one element using this class?

File: views/policies.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        els[84].click()
+        els[85].click()
+        els[86].click()
+        els[87].click()
+
+    def __assign_middleware_policy_datasource_events__(self, policy_name, els):
+        assert self.ui_utils.isTextOnPage('"{}" Event Assignments'.format(policy_name))
+
+        # EAP Datasource
+        els[88].click()
```
Comment (gbaufake) [inline views/policies.py]: Will the element always hold the position on HTML?

File: views/policies.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        els[89].click()
+        els[90].click()
+        els[91].click()
+
+    def __assign_middleware_policy_jdbc_events__(self, policy_name, els):
+        assert self.ui_utils.isTextOnPage('"{}" Event Assignments'.format(policy_name))
+
+        # EAP JDBC
+        els[92].click()
```
Comment (gbaufake) [inline views/policies.py]: Will the element always hold the position on HTML?

**Review by gbaufake — COMMENTED** [review]
@mattmahoneyrh 
Submitted my PR revision.

File: views/policies.py (hunk 2 above)
Comment (mattmahoneyrh) [reply views/policies.py]: yes, element positioning may change in the future, and in fact the entire workflow will be changing to a wizard type workflow. So, I will refactor completely when that happens.

**Comment by gbaufake** [comment]
Looks Good to me!
Merging!

**Pull request merged**